src/protocols/ethereum/components/TransferSendForm.vue

```diff
@@ -0,0 +1,314 @@
+<template>
+  <TransferSendFormBase
+    v-bind="$attrs"
+    :transfer-data="transferData"
+    :fee="numericFee"
+    :fee-symbol="ETH_SYMBOL"
+    :protocol="PROTOCOL_ETHEREUM"
+    :custom-title="$t('modals.send.sendAsset', { name: ETH_COIN_NAME })"
+    class="transfer-send-form"
+  >
+    <template #recipient>
+      <TransferSendRecipient
+        v-model.trim="formModel.address"
+        :placeholder="$t('modals.send.recipientPlaceholderEth')"
+        :errors="errors"
+        :protocol="PROTOCOL_ETHEREUM"
+        :validation-rules="{ address_eth: true }"
+        @openQrModal="openScanQrModal"
+      />
+    </template>
+
+    <template #amount>
+      <TransferSendAmount
+        v-model="formModel.amount"
+        :errors="errors"
+        :selected-asset="formModel.selectedAsset"
+        readonly
+        :protocol="PROTOCOL_ETHEREUM"
+        :validation-rules="{
+          ...+balance.minus(fee) > 0
+            ? { max_value: max.toString() }
+            : {},
+          enough_coin: [fee.toString(), ETH_SYMBOL],
+        }"
+        @asset-selected="handleAssetChange"
+      >
+        <template #label-after>
+          <BtnPlain
+            class="max-button"
+            :class="{ chosen: formModel?.amount?.toString() === max.toString() }"
+            @click="setMaxAmount"
+          >
+            {{ $t('common.max') }}
+          </BtnPlain>
+        </template>
+      </TransferSendAmount>
+    </template>
+
+    <template
+      #extra
+    >
+      <div
+        v-show="activeNetwork.type !== NETWORK_TYPE_TESTNET"
+      >
```

Comment:
This element does nothing. I suggest to put the `v-show` on the `DetailsItem` element.